concrete/src/Multilingual/Service/Extractor.php

```diff
@@ -72,6 +77,8 @@ public function extractTranslatableSiteStrings()
 
     public function getDynamicTranslations()
     {
+        $fh = \Core::make('helper/file');
+        \C5TL\Options::setTemporaryDirectory($fh->getTemporaryDirectory());
```

Comment:
> Is setting this something we should do in the default runtime?

I don't think so: concrete5-translation-library is only used by 0.0001% of calls, so I don't think we should autoload/initialize it for every call...